crates/lib/src/qpu/translation.rs

```diff
@@ -122,6 +122,12 @@ pub struct TranslationOptions {
 }
 
 impl TranslationOptions {
+    /// Get the backend used for translation
+    #[must_use]
+    pub fn get_backend(&self) -> Option<TranslationBackend> {
+        self.inner.translation_backend.clone()
+    }
```

Comment:
This name/signature is [more conventional](https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter).

```suggestion
    #[must_use]
    pub fn backend(&self) -> Option<&TranslationBackend> {
        self.inner.translation_backend.as_ref()
    }
```

Reply:
Hm, I glossed over the `.clone()` here. `TranslationBackend` is just an enum with unit members, i.e. a tiny numeric type. So it should be `Copy`.

Given this, I think it should probably return a value.

Reply:
....actually, `TranslationBackend` _is_ `Copy`, and `Option<T>` implements `Copy` IFF `T` does... so why do we need `clone()` here?

Reply:
`TranslationBackend` is _not_ copy.

Reply:
> This name/signature is [more conventional](https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter).

Kalan has often asked that getters be prefixed with `get_`, but happy to follow the Rust style guide.

Reply:
Is it a different `TranslationBackend` than this one? https://github.com/rigetti/qcs-sdk-rust/pull/373/files#diff-e83cc7a96cad7becb1c3df31f77635e277497be983a6dd80c3546ca22fade877R94

Reply:
...oh, I see, it's the GRPC type.